src/me/mrCookieSlime/Slimefun/api/TickerTask.java

```diff
@@ -238,14 +251,19 @@ public void info(CommandSender sender) {
 			tellraw.addText("   &7&oHover for more Info");
 			StringBuilder hover = new StringBuilder();
 			int hidden = 0;
-			
-			for (Map.Entry<String, Long> entry: chunkTimings.entrySet()) {
+
+			Map<String, Long> timings = chunkTimings.entrySet().stream()
+                    .sorted(Map.Entry.comparingByValue(Comparator.reverseOrder()))
+                    .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue, (e1, e2) -> e1, LinkedHashMap::new));
```

Comment:
If you care about it being sorted and need the entries themselves anyway, why even collect this as a map and not a List of Entries?